es/es-server/src/main/java/org/elasticsearch/common/lucene/uid/PerThreadIDVersionAndSeqNoLookup.java

```diff
@@ -108,7 +108,28 @@ public DocIdAndVersion lookupVersion(BytesRef id, LeafReaderContext context)
             if (versions.advanceExact(docID) == false) {
                 throw new IllegalArgumentException("Document [" + docID + "] misses the [" + VersionFieldMapper.NAME + "] field");
             }
-            return new DocIdAndVersion(docID, versions.longValue(), context.reader(), context.docBase);
+            final long seqNo;
+            final long term;
+            if (loadSeqNo) {
+                NumericDocValues seqNos = context.reader().getNumericDocValues(SeqNoFieldMapper.NAME);
+                // remove the null check in 7.0 once we can't read indices with no seq#
```

Comment:
Does this already apply for us? (Do all indices created under CrateDB 3.x have seqNos?)